samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java

```diff
@@ -38,11 +38,11 @@ public class PurgeUserEvent {
 
   public static void main(String[] args)
       throws IOException, ExecutionException, InterruptedException {
-    // TODO(developer): Replace these variables before running the sample.
-    String projectId = ServiceOptions.getDefaultProjectId();
+    // TODO: Set projectId to your Google Cloud Platform project ID.
```

Comment:
nit: do we want to replace TODO with TODO(developer)

Reply:
Replaced TODO with TODO(developer)